app/models/concerns/searchable.rb

```diff
@@ -64,6 +64,32 @@ def self.full_search(query)
       __elasticsearch__.search(query.build_query(highlighted_fields, fuzzy_fields, exact_fields))
     end
 
+    def self.search_for_businesses_in_batches(search_query, size = 1000)
```

Comment:
This should work for all models. Also suggest setting the default for `size` as a constant in the class so it could be overridden on a per-class basis (depending on the index size for each model).

```suggestion
    def self.search_in_batches(search_query, size = 1000)
```